arches/app/media/js/views/components/resource-report-abstract.js

```diff
@@ -69,6 +69,7 @@ define([
                         self.report({
                             'template': responseJson.template,
                             'report_json': responseJson.report_json,
+                            'resourceid': responseJson.resourceid
```

Comment:
In 6.2 I don't see a `resourceid` property on the responseJson object. Is it possible there were some backend changes not included in this PR? If not, it does look like there is a `responseJson.report_json.resourceinstanceid`

<img width="572" alt="image" src="https://user-images.githubusercontent.com/1877663/185717399-3b7cc37a-d582-4ff1-9e5b-bef130dd05d9.png">